js/views/audio_player.js

```diff
@@ -603,7 +633,7 @@ define(['jquery'],function($) {
         }
         
         var MAX_SEEK_RETRIES = 10;
-        var _seekedEvent1 = _iOS ? "canplaythrough" : "seeked"; //"progress"
+        var _seekedEvent1 = "seeked";
```

Comment:
This reminds me of this bug report (Android Chromium webview breaking changes):
https://github.com/readium/readium-js-viewer/issues/681

...I really hope that one day no audio-related code tweaks are needed for modern iOS (thank you BN for the PR, it looks like based on your experience the "seeked" event now works), but on Android the practical reality is that Readium must support older version (e.g. KitKat 4.4) where some tweaks are still required :(